src/Workspaces/Core/Portable/Utilities/ReferenceCountedDisposable.cs

```diff
@@ -0,0 +1,263 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Roslyn.Utilities
+{
+    /// <summary>
```

Comment:
It's fine like this. Amending would unnecessarily complicate things at this point.